### PR TITLE
Add schema for effective privileges, used in a recent addition to the…

### PR DESCRIPTION
**Pull request by mburbidgadobe**
… Platform API.

Please link to the issue #…

### CODE REVIEW

File: schemas/external/repo/effectiveprivileges.example1.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "*": ["read", "write"],
+    "api:ac": ["read"]
```
Comment (trieloff) [inline schemas/external/repo/effectiveprivileges.example1.json]: `api` is an unknown namespace. Use `repo` instead.

Comment (trieloff) [reply schemas/external/repo/effectiveprivileges.example1.json]: Or introduce the namespace (probably replacing `repo`)

File: schemas/external/repo/effectiveprivileges.schema.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  "$id": "https://ns.adobe.com/xdm/base/effectiveprivileges",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Effective Privileges",
+  "description": "Schema for the Effective Privileges resource",
+  "type": "object",
+  "definitions": {
+    "effectiveprivileges": {
+      "type": "object",
+      "properties": {
+        "*": { "$ref": "#/definitions/privilege" }
```
Comment (trieloff) [inline schemas/external/repo/effectiveprivileges.schema.json]: I know `*` is a valid JSON name, but it still feels odd to me. @kstreeter @lrosenthol WDYT?

**Comment by trieloff** [comment]
@mburbidgadobe please also take care of the failing build. `npm run lint` and `npm test` need to succeed.

File: schemas/external/repo/effectiveprivileges.schema.json (hunk 2 above)
Comment (mburbidgadobe) [reply schemas/external/repo/effectiveprivileges.schema.json]: "*" was suggested by Felix. It does accurately described the property. Alternative might be "default".

**Comment by mburbidgadobe** [comment]
I think all tests are succeeding now.

File: schemas/external/repo/effectiveprivileges.schema.json (hunk 2 above)
Comment (trieloff) [reply schemas/external/repo/effectiveprivileges.schema.json]: Ok, then we'll roll with it.

**Pull request merged**